apps/web/pages/cancel/[uid].tsx

```diff
@@ -278,10 +286,13 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   });
 
   let recurringInstances = null;
-  if (booking.eventType?.recurringEvent && allRemainingBookings === "true") {
+  if (booking.eventType?.recurringEvent && allRemainingBookings) {
     recurringInstances = await prisma.booking.findMany({
       where: {
         recurringEventId: booking.recurringEventId,
+        startTime: {
+          gte: new Date(),
+        },
```

Comment:
This was missing here too, we only need to cancel remaining, not all of them, so we should show the remaining ones that are going to be canceled should the user opts to cancel all remaining events.